source/style-guide/information-types/concept.txt

```diff
@@ -0,0 +1,101 @@
+======================
+Concept Page Prototype
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/info-typing/fact-general-guidance.rst
+
+A concept page helps a user learn about a concept or new feature. The page intends to 
+answer the following questions:
+
+- "What is this?"
+
+- "Why do I care?"
+
+- "How does this work?"
+
+The audience includes users who have never used a particular feature or
+command.
+
+The title is typically a noun or noun phrase. 
+
+A concept page begins with a brief introduction that summarizes its
+contents, often called a short description. The short description
+explains a little about the concept and why the audience should care
+about it.
+
+Command Syntax
+--------------
+
+Optional.
+
+If it's appropriate, you may include a code-block with a copyable
+prototype to familiarize users with command syntax.
+
+.. code-block:: javascript
+
+   db.collection.<method>( {
+      /* command fields */
+   } )
+
+After the code block, provide a link to the corresponding
+:ref:`reference <meta-reference-prototype>` or :ref:`task
+<meta-task-prototype>` page to provide readers with complete examples
+and behavior details. When possible, single-source code examples and
```

Comment:
```suggestion
and behavior details. When possible, use single-source code examples and
```
Small typo. Also, can you be more specific about what you mean by "single-source code examples"? Does this mean use the same code examples of object models across all pages?

Reply:
In the previous draft, "single-source" was being used as a verb, but I agree this could be made clearer.

What we mean here is to use includes to use the same examples across multiple pages. I will try to clarify this point.